CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [22.1.0]
+### Added
+- Add method to check and update collectible ownership state ([#664](https://github.com/MetaMask/controllers/pull/664))
+- Add finish to EIP-1559 v2 gas estimate code ([#660](https://github.com/MetaMask/controllers/pull/660))
```

Comment:
I think you can merge this one with the one that's on line 15 now as well, as it was related to that change.